composer.json

```diff
@@ -16,13 +16,13 @@
     },
 
     "require-dev": {
-        "behat/mink-goutte-driver": "^1.1",
-        "guzzlehttp/guzzle": "^6.3",
-        "behat/mink-selenium2-driver": "^1.6",
-        "atoum/atoum": "^4.0",
-        "fabpot/goutte": "^3.2",
-        "phpunit/phpunit": "^9.5",
-        "atoum/stubs": "^2.6"
+        "behat/mink-goutte-driver": "^1.3 || ^2.0",
```

Comment:
Deprecated. We should use `behat/mink-browserkit-drive` instead. It should resolve PHPStan errors at the same time.